msal/src/main/java/com/microsoft/identity/client/KnownAuthorities.java

```diff
@@ -0,0 +1,34 @@
+package com.microsoft.identity.client;
```

Comment:
Need to license info (can follow in subsequent PR)

Reply:
Missed this one I guess.  Sounds good.

Reply:
Done.